src/test/java/com/google/edith/UserInsightsTest.java

```diff
@@ -217,28 +230,28 @@ public void createJson_withUnkownUser_returnsDefaultJson() {
 
   /**
    * Creates a list of Item keys
+   *
    * @param startIndex the number of the first element in the list
    * @param endIndex the number of the last element in the list
-   * @return a list of keys with a number assigned to each of them 
-   *         "item0, item1... itemN"
+   * @return a list of keys with a number assigned to each of them "item0, item1... itemN"
    */
   private List<Key> createTestKeyList(int startIndex, int endIndex) {
     List<Key> items = new ArrayList<>();
-    for(int i = startIndex; i < endIndex; i++) {
+    for (int i = startIndex; i < endIndex; i++) {
       items.add(KeyFactory.createKey("Item", "Item" + i));
     }
     return items;
```

Comment:
Can use streams here:

```java
private List<Key> createTestKeyList(int start, int count) {
  return IntStream.range(start, start + count)
    .map(i -> KeyFactory.createKey("Item", "Item" + i))
    .collect(Collectors.toList());
}
```

The benefit being immutability as `ArrayList` is a mutable type.

Reply:
Changes on PR #35